### PR TITLE
Improve build

### PR DESCRIPTION
**Pull request by KorDum**
Fix build config
Added README file with short instruction for build mod
Added Forge License
Add eclipse folder to gitignore

### CODE REVIEW

**Comment by marcin212** [comment]
What do you think @mezz?

File: build.gradle [hunk 1]
```diff
@@ -18,34 +18,34 @@ buildscript {
 apply plugin: 'forge'
 
 version = "2.0-pre15"
-group= "binnie" // http://maven.apache.org/guides/mini/guide-naming-conventions.html
+group = "binnie" // http://maven.apache.org/guides/mini/guide-naming-conventions.html
 archivesBaseName = "binnie-mods"
+sourceCompatibility = JavaVersion.VERSION_1_7
+targetCompatibility = JavaVersion.VERSION_1_7
```
Comment (mezz) [inline build.gradle]: These should be Java 8, not 7.

File: MinecraftForge-License.txt [hunk 2]
```diff
@@ -0,0 +1,70 @@
+Minecraft Forge Public Licence
```
Comment (mezz) [inline MinecraftForge-License.txt]: Forge license is not required, we do not ship Forge.

File: MinecraftForge-Credits.txt [hunk 3]
```diff
@@ -0,0 +1,26 @@
+* Eloraam *
```
Comment (mezz) [inline MinecraftForge-Credits.txt]: Forge credits are not required, we are not Forge.

**Review by mezz — CHANGES_REQUESTED** [review]
The script changes look good but some things need to change before it can be pulled.

**Comment by KorDum** [comment]
Done. How can I assignment to reviewer automatically?

**Comment by mezz** [comment]
Just leave a comment like you just did, anyone reviewing will get a notification. I'll take a look.

**Review by mezz — APPROVED** [review]

**Comment by mezz** [comment]
Looks good, tested and it set up a new 1.7.10 project correctly.

**Pull request merged**